ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidatorFactory.scala

```diff
@@ -50,17 +47,16 @@ object BatchedSubmissionValidatorFactory {
   def readerAndCommitStrategyFrom[LogResult](
       ledgerStateOperations: LedgerStateOperations[LogResult],
       keySerializationStrategy: StateKeySerializationStrategy = DefaultStateKeySerializationStrategy,
-      ledgerDataExporter: LedgerDataExporter = LedgerDataExporter())(
-      implicit executionContext: ExecutionContext)
+  )(implicit executionContext: ExecutionContext)
     : (DamlLedgerStateReader, CommitStrategy[LogResult]) = {
     val ledgerStateReader = DamlLedgerStateReader.from(
       new LedgerStateReaderAdapter[LogResult](ledgerStateOperations),
-      keySerializationStrategy)
-    val commitStrategy =
-      new LogAppendingCommitStrategy[LogResult](
-        ledgerStateOperations,
-        keySerializationStrategy,
-        ledgerDataExporter)
+      keySerializationStrategy,
+    )
+    val commitStrategy = new LogAppendingCommitStrategy[LogResult](
+      ledgerStateOperations,
+      keySerializationStrategy,
```

Comment:
Why wouldn't the `LogAppendingCommitStrategy` support ledger exports?

Reply:
It does, but now they get pushed through at the method level, not in the constructor, so we can avoid mutation.